### DISCUSSION TITLE
How create tabs like vuepress docs?

### DISCUSSION THREAD

**Discussion by jonathanbergson** [vuepress/core#581] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6494405/146760110-29767bd7-07c5-41e4-889b-d4d7dbc0cb27.png)

**Comment by qcyblm** [comment, marked as answer]
https://v2.vuepress.vuejs.org/reference/default-theme/markdown.html#custom-containers
Example 3 (code group alias)

**Reply by jonathanbergson** [reply]
thanks for the help
